nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -138,63 +135,59 @@
+- [bitmagnet](https://bitmagnet.io/), a self-hosted BitTorrent indexer, DHT crawler, content classifier and torrent search engine with WebUI, GraphQL API and Servarr stack integration. Available as [services.bitmagnet](options.html#opt-services.bitmagnet.enable).
 
 - [Wakapi](https://wakapi.dev/), a time tracking software for programmers. Available as [services.wakapi](#opt-services.wakapi.enable).
 
 - [foot](https://codeberg.org/dnkl/foot), a fast, lightweight and minimalistic Wayland terminal emulator. Available as [programs.foot](#opt-programs.foot.enable).
 
-- [ToDesk](https://www.todesk.com/linux.html), a remote desktop applicaton. Available as [services.todesk.enable](#opt-services.todesk.enable).
+- [ToDesk](https://www.todesk.com/linux.html), a remote desktop application. Available as [services.todesk](#opt-services.todesk.enable).
 
-- [Dependency Track](https://dependencytrack.org/), an intelligent Component Analysis platform that allows organizations to identify and reduce risk in the software supply chain. Available as [services.dependency-track](option.html#opt-services.dependency-track).
+- [Dependency Track](https://dependencytrack.org/), an intelligent Component Analysis platform that allows organizations to identify and reduce risk in the software supply chain. Available as [services.dependency-track](option.html#opt-services.dependency-track.enable).
 
 - [Immich](https://github.com/immich-app/immich), a self-hosted photo and video backup solution. Available as [services.immich](#opt-services.immich.enable).
 
-- [saunafs](https://saunafs.com) Distributed POSIX file system. Available as [services.saunafs](options.html#opt-services.saunafs).
+- [saunafs](https://saunafs.com), a distributed POSIX file system. Available as [services.saunafs](options.html#opt-services.saunafs.enable).
 
-- [obs-studio](https://obsproject.com/), Free and open source software for video recording and live streaming. Available as [programs.obs-studio.enable](#opt-programs.obs-studio.enable).
+- [obs-studio](https://obsproject.com/), a free and open source software for video recording and live streaming. Available as [programs.obs-studio](#opt-programs.obs-studio.enable).
 
-- [Veilid](https://veilid.com), a headless server that enables privacy-focused data sharing and messaging on a peer-to-peer network. Available as [services.veilid](#opt-services.veilid.enable).
+- [Veilid](https://veilid.com), a privacy-focused, headless server for data sharing and messaging on a peer-to-peer network. Available as [services.veilid](#opt-services.veilid.enable).
 
-- [Fedimint](https://github.com/fedimint/fedimint), a module based system for building federated applications (Federated E-Cash Mint). Available as [services.fedimintd](#opt-services.fedimintd).
+- [Fedimint](https://github.com/fedimint/fedimint), a module based system for building federated applications (Federated E-Cash Mint). Available as [services.fedimintd](#opt-services.fedimintd.enable).
```

Comment:
> bad local reference, id #opt-services.fedimintd.enable not known

https://github.com/NixOS/nixpkgs/actions/runs/11451771665/job/31861545904?pr=346059#step:5:137

Getting this error because [the appendix page for fedimintd](https://nixos.org/manual/nixos/unstable/options#opt-services.fedimintd) doesn't have the `.enable` bit.

A few loosely related notes:
1. Is there no convention for this? Having or not having `.enable` (AFAIK that appendix is auto generated, so it should have some consistency)
2. IIRC the hyperlinks for module options are auto-filled by nixos-render-docs, so we probably don't need to specify that anywhere? Would need confirmation on this.

Reply:
> Having or not having .enable

As long as the `.enable` option exists, yes. As it turns out, fedimintd just [doesn't have one](https://github.com/NixOS/nixpkgs/blob/729d9ddec137ab399cfdab567072896de8a0ccf8/nixos/modules/services/networking/fedimintd.nix#L183-L187)

> IIRC the hyperlinks for module options are auto-filled by nixos-render-docs

As in `[](#opt-services.foo.enable)`? That does work, but it would end up making the hyperlink the literal `services.foo.enable` rather than only `services.foo` -- and most modules won't have a single root `services.foo` option